webapps/src/main/webapp/vue-app/taskCommentsDrawer/components/TaskCommentEditor.vue

```diff
@@ -199,6 +200,15 @@ export default {
         this.taskCommentAttachmentsEdited = true;
       }
     },
+    getSpaceUrlFromProjectParticipator() {
+      const participator = this.task.status.project.participator;
+      if (participator && participator.length > 0) {
+        const projectParticipator = participator.find((element) => element.startsWith('member:/spaces/'));
+        if (projectParticipator) {
+          return projectParticipator.substring(15);
+        }
+      }
+    },
```

Comment:
Please add the variable into a computed and you can test in `/space/` only without `member:` to allow better extensibility and maintainability (by example, when we add `*:/spaces/XYZ` instead of `member:/spaces/XYZ`, this should work as well)

Besides, in the proposed change, we will avoid a Null Pointer Exception when the task isn't added to a project yet.

```suggestion
    computed: {
      taskSpaceUrl() {
        const projectMembers = this.task?.status?.project?.participator;
        if (projectMembers?.length) {
          const projectSpace = projectMembers.find(member => member.includes('/spaces/'));
          if (projectSpace) {
            return projectSpace.split('/spaces/')[1];
          }
        }
        return null;
      },
    }
```

Reply:
Thanks for the feedback. Working on it.